desktop/cmp/input/DateInput.js

```diff
@@ -123,7 +123,10 @@ DateInput.propTypes = {
     ]),
 
     /** Boundary for calendar popover, as per Blueprint docs. Defaults to viewport. */
-    popoverBoundary: PT.oneOf(['scrollParent', 'viewport', 'window']),
+    popoverBoundary: PT.oneOf(['scrollParent', 'viewport', 'window', PT.element]),
```

Comment:
BluePrint docs are wrong here.  Popper will accept any Element as well.
Popper Docs: https://popper.js.org/docs/v1/#modifiers..preventOverflow.boundariesElement
Blueprint use of boundary: https://github.com/palantir/blueprint/blob/d3b72d4d81efa89724567a907b879d0b428c080d/packages/core/src/components/popover/popover.tsx#L453